requirements.txt

```diff
@@ -1,5 +1,6 @@
 checklist==0.0.10
 spacy==2.2.4
+pyinflect==0.5.1
```

Comment:
We recommend creating a separate requirements.txt in the same transformation folder so that the main requirements.txt is not changed often (which might change the version of the dependency for specific transformation)